articles-java-jee/src/main/java/com/ibm/cns/articles/boundary/ArticlesService.java

```diff
@@ -0,0 +1,129 @@
+package com.ibm.cns.articles.boundary;
+
+import com.ibm.cns.articles.control.DataAccess;
+import com.ibm.cns.articles.control.MPConfigured;
+import com.ibm.cns.articles.entity.Article;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+import javax.annotation.PostConstruct;
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+@ApplicationScoped
```

Comment:
Hi Adam,
I know it was not introduced by you but is @ApplicationScoped necessary?
BTW: Thx for your refactoring :-)